admin/class-yoast-notification.php

```diff
@@ -355,6 +355,15 @@ public function render() {
 		return '<div ' . implode( ' ', $attributes ) . '>' . $message . '</div>' . PHP_EOL;
 	}
 
+	/**
+	 * Get the message for the notification.
+	 *
+	 * @return string The message.
+	 */
+	public function get_message() {
+		return wpautop( $this->message );
```

Comment:
wpautop turns line break to paragraphs, not just wrapping the text in p tag. It also matches the way notifications are rendered in the old dashboard and since we are not addressing that in this stage, I'd say lets keep it.